liferay-workspace/modules/blade.language.web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -0,0 +1,21 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<p>
+	<b><liferay-ui:message key="blade_language_web_LanguageWebPortlet.caption" /></b>
```

Comment:
Lets show 3 captions here.

1) one caption load from blade.language module solely
2) one caption that is using a key that is defined in both language and langauge.web to show that language.web overrides it
3) one caption that only exists in language.web